mapie/tests/test_classification.py

```diff
@@ -1865,8 +1869,7 @@ def test_get_true_label_position(
     y_pred_proba = y_true_proba_place[1]
     place = y_true_proba_place[2]
 
-    mapie = MapieClassifier(random_state=random_state)
-    found_place = mapie._get_true_label_position(y_pred_proba, y_true)
+    found_place = get_true_label_position(y_pred_proba, y_true)
```

Comment:
This test should be implemented directly in test_utils_classification_conformity_scores